tests/common/test_io.py

```diff
@@ -80,3 +92,64 @@ def test_attach_stderr_handler():
     assert c.stdout == ""
     assert "test message" in c.stderr
     assert debug_message in c.stderr
+
+
+def test_console_handler():
```

Comment:
Please add a comment to explain why this test exists. Please do this for the other tests too.